### PR TITLE
Used redux in react components

### PR DESCRIPTION
**Pull request by JacintaNgwu**
- Used store, actions, and reducers in React.
- Connected React and Redux.
- Used useSelector to consume the state.
- Imported Redux useDispatch.
-  Added functionality for the "Remove" button click
- Added functionality for the "Check status" button click

### CODE REVIEW

**Comment by jerryowusu** [comment]
Hello @JacintaNgwu

**Comment by jerryowusu** [comment]
Kindly edit and update the Future Features of your readme file to reflect the current project. :+1:

File: src/pages/Categories.js [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
+    dispatch(checkStatus());
+  };
+  return (
+    <div className="categories">
+      <button type="button" className="update" onClick={handleClick}>
+        Check Status
+      </button>
+    </div>
+  );
+};
```
Comment (jerryowusu) [inline src/pages/Categories.js]: The page should display `Under construction` when the button is clicked, however, nothing happens you click on the button. Kindly update to reflect this requirement. :wink:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @JacintaNgwu , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:


almost there

## Highlights
No linter errors :+1: 
Great effort so far :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @JacintaNgwu, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good description :heavy_check_mark:
 
## [Optional] suggestions:
* [ ]  Please check my inline comments.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**